packages/api-catalog-service/src/types.d.ts

```diff
@@ -100,6 +100,10 @@ interface NamespaceInput {
     updatedBy?: Maybe<Scalars[ 'String' ]>;
 }
 
+interface ListNamespacesArgs {
+    limit?: Maybe<Scalars['Int']>;
```

Comment:
What is the benefit of using the `Scalers` interface. Isn't it another abstraction?

Reply:
updated